test/staking/candidates.staking.test.ts

```diff
@@ -0,0 +1,252 @@
+/*
+ *
+ * @group parallel
```

Comment:
Should we add `stacking `group here?

Reply:
If it fits on regular parallel executions, I think we can skip it